test/pythonml/test_context.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import os
+import re
+import unittest
+from unittest import mock
+
+
+class TestContext(unittest.TestCase):
+    """
+    Unit tests for the methods in the Context class defined in the settings.py module
+    """
+
+    def setUp(self):
+        with open('fixtures/settings.py', 'r') as file:
+            unmodified_settings_file = file.readlines()
```

Comment:
I'd recommend calling this `settings_template` to be a little more explicit about what it is

Reply:
Done